update.sh

```diff
@@ -31,6 +31,11 @@ lock_file build.lck || {
     echo "$(date +%s) already locked" >>build.lck.log
     exit 1
 }
+# activate Python venv on more recent releases:
+if [ ${DISTRIBUTION_CODENAME} = 'mantic' ]; then
```

Comment:
Weren't we talking noble?!

Reply:
Sorry. Did a push without commiting from the old PR. Should be good now.